Mathlib/CategoryTheory/Category/Cat/Colimit.lean

```diff
@@ -0,0 +1,37 @@
+/-
+Copyright (c) 2025 Mario Carneiro and Emily Riehl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro, Emily Riehl
+-/
+import Mathlib.AlgebraicTopology.SimplicialSet.NerveAdjunction
+import Mathlib.CategoryTheory.Monad.Limits
+
+
+/-!
+# The category of small categories has all small colimits.
+
+A reflective subcategory inherits any colimits present in the ambient category, constructed
+by applying the reflector. Thus the fully faithful nerve embedding into simplicial sets and
+its left adjoint provide a construction of colimits in the category of small categories.
```

Comment:
```suggestion
In this file, the existence of colimits in `Cat` is deduced from the existence of
colimits in the category of simplicial sets. Indeed, `Cat` identifies to a reflective
subcategory of the category of simplicial sets (see `AlgebraicTopology.SimplicialSet.NerveAdjunction`),
so that colimits in `Cat` can be computed by passing to nerves, taking the colimit
in `SSet` and finally applying the homotopy category functor `SSet ⥤ Cat`. 
```